weaviate/util.py

```diff
@@ -772,3 +777,34 @@ def _type_request_response(json_response: Any) -> Optional[Dict[str, Any]]:
         return None
     assert isinstance(json_response, dict)
     return json_response
+
+
+def _decode_json_response_dict(
+    response: requests.Response, location: str
+) -> Optional[Dict[str, Any]]:
+    if response is None:
+        return None
+
+    if response.status_code == 200:
```

Comment:
This is something I wanted to change a long time ago. I saw that most libraries to not check exactly for a single status code but for a valid one which is in range 200-299, so maybe we can do it now:
```python
if response.status_code >= 200 and response.status_code < 300:
```

This is not critical.

Reply:
looks good! :)